docs/plugin_development.rst

```diff
@@ -352,48 +427,57 @@ After ``cookicutter`` finishes its job, you'll get the following tree structure
 
-.. code-block:: bash
-
-   clean                remove all build, test, coverage and Python artifacts
-   clean-build          remove build artifacts
-   clean-pyc            remove Python file artifacts
-   clean-test           remove test and coverage artifacts
-   test                 run tests quickly with the default Python
-   docs                 generate Sphinx HTML documentation, including API docs
-   servedocs            compile the docs watching for changes
-   release              package and upload a release
-   dist                 builds source and wheel package
-   install              install the package to the active Python's site-packages
-   develop              install the package to the active Python's site-packages
-
-
-* ``setup.py``, which helps you to install, package and distribute your plugin with ``setuptools``, the standard for distributing Python Modules.
-  On this file the ``entry_points`` parameter of ``setup`` is quite important, as it is the one that allows Spyder to identify this package as a plugin, and to know how to access its functionalities.
+  .. code-block:: text
 
+     clean                remove all build, test, coverage and Python artifacts
+     clean-build          remove build artifacts
+     clean-pyc            remove Python file artifacts
+     clean-test           remove test and coverage artifacts
+     test                 run tests quickly with the default Python
+     docs                 generate Sphinx HTML documentation, including API docs
+     servedocs            compile the docs watching for changes
+     release              package and upload a release
+     dist                 builds source and wheel package
+     install              install the package to the active Python's site-packages
+     develop              install the package to the active Python's site-packages
 
+* ``setup.py``.
+  It helps you install, package and distribute your plugin with Setuptools, the standard for distributing Python packages.
+  In this file the ``entry_points`` parameter of ``setup`` is quite important, as it is what allows Spyder to identify this package as a plugin, and know how to access its functionality.
 
-The ``spyder-pomodoro-timer`` folder has the name you introduced when running ``cookiecutter``. Inside this you'll see a folder called ``spyder``, where we will place the code of our plugin.
+The ``spyder-pomodoro-timer`` folder has the same name you entered when you ran ``cookiecutter``.
+Inside it you will see a folder called ``spyder``, where we will place the code of our plugin.
 
 In the ``spyder`` directory you'll find the following files:
 
-* ``api.py``: where the functionality of the plugin is exposed to the rest of Spyder. That would allow additional functionality to be added from other plugins.
+* ``api.py``.
+  Exposes the functionality of the plugin to the rest of Spyder.
+  That allows adding additional functionality from other plugins.
```

Comment:
```suggestion
  Exposes the functionality of the plugin to the rest of Spyder, allowing other plugins to build on top of it in turn.
```

I found the rewritten form confusing and less clear, as well as more choppy, but I think I've been able to restore that.